modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/UniqueCompositeKey.tsx

```diff
@@ -365,7 +367,14 @@ export function UniqueCompositeKey({
 
 		setBuilderScreenItems(newBuilderScreenItems);
 		setModalSelectObjectFieldsItems(newModalSelectObjectFieldsItems);
-		setMultipleSelectOptions(newMultipleSelectOptions);
+
+		setMultipleSelectOptions([
+			{
+				children: newMultipleSelectOptionsChildren,
+				label: '',
+				value: 'field',
```

Comment:
Hey @larissacribeiro, what do you think to rename 'field' to `'objectFields' `. 
To use object prefix and plural, since we can see more than one object field in this `MultipleSelect` options.